src/plugins/data_view_management/public/components/edit_index_pattern/index_header/index_header.tsx

```diff
@@ -63,10 +83,23 @@ export const IndexHeader: React.FC<IndexHeaderProps> = ({
             iconType="pencil"
             aria-label={editAriaLabel}
             data-test-subj="editIndexPatternButton"
+            color="primary"
           >
             {editTooltip}
           </EuiButton>
         ),
+        <EuiToolTip content={<p>{refreshTooltip}</p>}>
+          <EuiButtonEmpty
```

Comment:
Can we hide the button if caching is disabled?

Reply:
We could, although I think I prefer to keep it since there's no update mechanism for the page. Not a strong opinion. Perhaps its best to discuss it with the team to see what people think.